packages/ui-settings/src/defaults/chains.ts

```diff
@@ -1,17 +1,15 @@
 // Copyright 2017-2020 @polkadot/ui-settings authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
+import { all } from '@polkadot/networks';
+
 type ChainDef = string[];
 
-const chains: Record<string, ChainDef> = {
-  kusama: [
-    '0x3fd7b9eb6a00376e5be61f01abb429ffb0b104be05eaff4d458da48fcd425baf', // Kusama CC1
-    '0xe3777fa922cafbff200cadeaea1a76bd7898ad5b89f7848999058b50e715f636', // Kusama CC2
-    '0xb0a8d493285c2df73290dfb7e61f870f17b41801197a149ca93654499ea3dafe' // Kusama CC3
-  ],
-  polkadot: [
-    '0x91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3' // Polkadot CC1
-  ]
-};
+// Kusama is the only network to have several genesis hashes
+const kusamaHashes = all.filter(({ genesisHash }) => genesisHash && genesisHash.length > 1)?.[0].genesisHash;
+
+const chains: Record<string, ChainDef> = (!!kusamaHashes?.length && {
+  kusama: kusamaHashes
+}) || {};
```

Comment:
Rather do it generically - we will never remember about this one.

```js
// never all, known has all these with genesis
import known from '@polkadot/networks';

const chains = known
  .filter(({ genesisHash }) => genesisHash.length > 1)
  .reduce((chains, { network, genesisHash }) => ({ ...chains, [network]: genesisHash }), {});
```

Reply:
ah wonderfull, thanks